### PR TITLE
feat: add tsuru-cli

### PR DESCRIPTION
**Pull request by devarthurribeiro**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: src/tsuru.ts [hunk 1]
```diff
@@ -0,0 +1,767 @@
...
+      name: "plan",
+      description: "Plan to be used by the app",
+      generators: tsuruGenerators.plans,
+    },
+  },
+};
+
+const completionSpec: Fig.Spec = {
+  name: "tsuru",
+  description: "",
```
Comment (SeparateRecords) [inline src/tsuru.ts:119]: (actually just noticed this too, can you add a description here?)

File: src/tsuru.ts [hunk 2]
```diff
@@ -0,0 +1,767 @@
+import { version } from "os";
```
[the remark this answers is not in the document]
Comment (devarthurribeiro) [reply src/tsuru.ts]: done!

File: src/tsuru.ts (hunk 1 above)
Comment (devarthurribeiro) [reply src/tsuru.ts]: done!

**Review by SeparateRecords — APPROVED** [review]
Awesome, thank you @devarthurribeiro!

**Pull request merged**